components/InformationBar.tsx

```diff
@@ -134,6 +155,12 @@ export default function InformationDrawer(props: { requirementsDisplay: Requirem
         </DrawerHeader>
         {open ? <Divider /> : undefined}
         <TabPanel value={value} index={0} key={0}>
+          <Requirement
+            name={open ? "Credit Total: " + getTotalCredits(props.semesters) + " out of 120" : "Credits"}
+            percentage={(getTotalCredits(props.semesters) / 120) * 100}
+            digits={10}
```

Comment:
This puts one decimal whether it's open or closed. 
Try` {open ? 10 : 1}`

Reply:
This seems to have fixed it